### PR TITLE
About us section grid fixed

### PR DESCRIPTION
**Pull request by tanishq-arya**
# Related Issue
Fixes: #215 

# Prosposed Changes
- Use of grid template areas
- Media queries

# Additional Info
- any additional information or context

# Screenshots

Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **
image

### CODE REVIEW

**Comment by garg3133** [comment]
@tushhr Can you please review this?

File: static/new_home/style.css [hunk 1]
```diff
@@ -468,31 +469,59 @@ header
...
+
 .vision{
   display: grid;
   grid-template-columns: 1fr 1fr;
+  grid-template-rows: auto;
+  grid-template-areas: "left right";
+  margin-bottom: 2em;
   grid-column-gap: 2em;
   padding: 20px 0;
+  margin-bottom: 1em;
```
Comment (garg3133) [inline static/new_home/style.css:481]: You've declared margin-bottom twice.

**Review by garg3133 — COMMENTED** [review]
Everything looks good! Just change a few things as stated below.

Also, what's the use of grid-template-rows here? It doesn't seem to do anything!

One more thing is that the image is overflowing on small screens (<400px), so you might want to fix that too (just reduce the size of image for very small screen sizes).

**Comment by tanishq-arya** [comment]
> Everything looks good! Just change a few things as stated below.
> 
> Also, what's the use of grid-template-rows here? It doesn't seem to do anything!
> 
> One more thing is that the image is overflowing on small screens (<400px), so you might want to fix that too (just reduce the size of image for very small screen sizes).

It is for the height of the rows, it is set to auto so for example if one row is 100px other is 300px in height then it automatically gives them the appropriate height. And if I would add more divs they can be aligned in same grid like this.
image

File: static/new_home/style.css [hunk 2]
```diff
@@ -523,6 +522,12 @@ header
   }
 }
 
+@media screen and (max-width:600px) {
+  img {
```
Comment (garg3133) [inline static/new_home/style.css:526]: This changes the size of all the images on the page, whereas you just needed to change the size of images inside the about-us section.

File: static/new_home/style.css [hunk 3]
```diff
@@ -523,6 +522,12 @@ header
...
+    max-width: 275px;
```
Comment (garg3133) [inline static/new_home/style.css:527]: With this size too, some part of the images and the text below cuts out on small screens

**Review by garg3133 — COMMENTED** [review]
Looks good! Just a few things were wrong in the latest changes as mentioned below, which I have corrected myself.

Merging this PR into master 🎉 

Thanks, @tanishq-arya for your contribution 🚀

File: static/new_home/style.css (hunk 2 above)
Comment (tanishq-arya) [reply static/new_home/style.css:526]: Oh okay

File: static/new_home/style.css (hunk 3 above)
Comment (tanishq-arya) [reply static/new_home/style.css:527]: Should I make changes now as this is merged?

Comment (garg3133) [reply static/new_home/style.css:527]: No, I've made the changes myself before merging it.

**Pull request merged**